schema.json

```diff
@@ -15,7 +15,7 @@
       },
       {
         "type": "select",
-        "options": ["Small", "Medium", "Large"],
+        "options": ["Small", "Medium", "Large","xx-large","xxx-large"],
         "defaultValue": "Large",
         "key": "size",
         "label": "Size"
```

Comment:
Thanks for the PR

Reply:
Hopefully this is all that needed to be changed to add the added options. I am not sure how to test this.